README.md

```diff
@@ -609,23 +609,31 @@ Options:
 - `--summary` - Output summary in JSON format
 - (default) - Output full report in JSON format
 
+#### Backend Configuration
+
+The client requires the backend URL to be configured through the environment variable:
+
+- **Environment variable**: `TRUSTIFY_DA_BACKEND_URL=https://backend.url` (required)
+
+The application will fail to start if this environment variable is not set.
+
 #### Examples
 
 ```shell
 # Stack analysis with JSON output (default)
-java -jar trustify-da-java-client-cli.jar stack /path/to/pom.xml
+TRUSTIFY_DA_BACKEND_URL=https://backend.url java -jar trustify-da-java-client-cli.jar stack /path/to/pom.xml
```

Comment:
No need to add the env var everywhere. you can use a common export for it to make it cleaner